modules/core/ui/context/config/optional/block.block.farm_context.yml

```diff
@@ -0,0 +1,23 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - farm_ui_context
+  theme:
+    - gin
+id: farm_context
+theme: gin
```

Comment:
We'll need to figure out how to do this without an explicit dependency on Gin. We maintain the ability for downstream users to develop/use their own theme by encapsulating all Gin-specific stuff in `farm_ui_theme`.

Perhaps `farm_ui_theme` could install this block, and depend on `farm_ui_context`? :thinking: 

Then it would be the responsibility of downstream users with custom themes to add the block themselves, but at least all the other functionality of contexts would exist.